src/workerd/api/node/dns.h

```diff
@@ -88,10 +63,78 @@ class DnsUtil final: public jsg::Object {
 
   using ReverseCallback = jsg::Function<void(void)>;
   void reverse(jsg::Lock& js, kj::String ip, ReverseCallback callback);
+};
+
+class DnsResolver final: public DnsResolverBase, public jsg::Object {
```

Comment:
I know you node people know these by heart but I would _love_ to see comments like:

// https://nodejs.org/api/dns.html#class-dnsresolver

on these classes.

Reply:
Definitely.